lab-andrew/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "lab-05-tcp-chat-server",
+  "version": "0.1.0",
+  "description": "wack-chat initial setup",
+  "main": "server.js",
+  "scripts": {
+    "test": "eslint",
+    "start": "node server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/andrewpeterwatson/lab-05-tcp-chat-server.git"
+  },
+  "author": "andrewpeterewatson@gmail.com",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/andrewpeterwatson/lab-05-tcp-chat-server/issues"
+  },
+  "homepage": "https://github.com/andrewpeterwatson/lab-05-tcp-chat-server#readme",
+  "devDependencies": {
```

Comment:
You'll need to add 'gulp' and 'eslint' to the devDependencies.  Gulp can't run without it, and if your "scripts" > "test" command uses 'eslint' the user will need that as well.